web/scripts/embed.dart

```diff
@@ -4,8 +4,8 @@
 
 // Currently not in use
 
-import 'package:dart_pad/embed/embed.dart' as embed;
-import 'package:dart_pad/polymer/polymer.dart';
+import '../../lib/embed/embed.dart' as embed;
+import '../../lib/polymer/polymer.dart';
```

Comment:
I suspect these changes were made by the analysis server's refactoring code? Agree that we want to prefer the `package:` references.